app/mailers/application_mailer.rb

```diff
@@ -9,15 +9,20 @@ def deliver
 
   class << self
     def mail_for_notification(notification)
-      if notification.notification_code == MAILBOXER_CODE_ACCESS_REQUEST
-        request = notification.notified_object
-        MapMailer.access_request_email(request)
-      elsif notification.notification_code == MAILBOXER_CODE_ACCESS_APPROVED
-        request = notification.notified_object
-        MapMailer.access_approved_email(request)
-      elsif notification.notification_code == MAILBOXER_CODE_INVITE_TO_EDIT
-        user_map = notification.notified_object
-        MapMailer.invite_to_edit_email(user_map.map, user_map.map.user, user_map.user)
+      case notification.notification_code
+        when MAILBOXER_CODE_ACCESS_REQUEST
+          request = notification.notified_object
+          MapMailer.access_request(request)
+        when MAILBOXER_CODE_ACCESS_APPROVED
+          request = notification.notified_object
+          MapMailer.access_approved(request)
+        when MAILBOXER_CODE_INVITE_TO_EDIT
+          user_map = notification.notified_object
+          MapMailer.invite_to_edit(user_map)
+        when MAILBOXER_CODE_MAP_MESSAGE
+        when MAILBOXER_CODE_MAP_STARRED
+        when MAILBOXER_CODE_TOPIC_ADDED_TO_MAP
+        when MAILBOXER_CODE_TOPIC_CONNECTED
```

Comment:
this is great. one thought i have about this is that this approach, where it's built into application_mailer, might be too limiting. I was naively hoping notifications would remain simple, but like 2 days after i merged mailboxer it was clear they wouldn't ahaha. i should have guessed that anyways...

but yeah we might need to change this approach later. just noting that.